crates/next-core/src/next_shared/webpack_rules/mod.rs

```diff
@@ -174,3 +236,62 @@ fn loader_runner_package_mapping() -> Result<Vc<ImportMapping>> {
     ])
     .cell())
 }
+
+#[turbo_tasks::value]
+struct ManuallyConfiguredBuiltinLoaderIssue {
+    glob: RcStr,
+    loader: RcStr,
+    disable_builtin_config_key: RcStr,
+    config_file_path: FileSystemPath,
+}
+
+#[turbo_tasks::value_impl]
+impl Issue for ManuallyConfiguredBuiltinLoaderIssue {
+    fn severity(&self) -> IssueSeverity {
+        IssueSeverity::Warning
+    }
+
+    #[turbo_tasks::function]
+    fn file_path(&self) -> Vc<FileSystemPath> {
+        self.config_file_path.clone().cell()
+    }
+
+    #[turbo_tasks::function]
+    fn stage(&self) -> Vc<IssueStage> {
+        IssueStage::Config.cell()
+    }
+
+    #[turbo_tasks::function]
+    fn title(&self) -> Vc<StyledString> {
+        StyledString::Line(vec![
+            StyledString::Text(rcstr!("Identified a likely manual configuration of ")),
+            StyledString::Code(self.loader.clone()),
+            StyledString::Text(rcstr!(" for paths matching ")),
+            StyledString::Code(self.glob.clone()),
+        ])
+        .cell()
+    }
+
+    #[turbo_tasks::function]
+    fn description(&self) -> Vc<OptionStyledString> {
+        Vc::cell(Some(
+            StyledString::Stack(vec![
+                StyledString::Text(rcstr!(
+                    "Next.js includes a built-in version of this loader that is configured \
+                     automatically. You may not need to configure this."
+                )),
+                StyledString::Line(vec![
+                    StyledString::Text(rcstr!("You can silence this warning by setting ")),
+                    StyledString::Code(self.disable_builtin_config_key.clone()),
+                    StyledString::Text(rcstr!(" in ")),
+                    StyledString::Text(self.config_file_path.path.clone()),
+                    StyledString::Text(rcstr!(" to ")),
+                    StyledString::Code(rcstr!("true")),
+                    StyledString::Text(rcstr!(" or ")),
+                    StyledString::Code(rcstr!("false")),
+                ]),
+            ])
+            .resolved_cell(),
+        ))
+    }
+}
```

Comment:
I tested against an example in the repo, adding a modified `next.config.ts` file. The warning looks like this:

![Screenshot 2025-08-28 at 6.05.41 PM.png](https://app.graphite.dev/user-attachments/assets/dc3708e1-6098-45ae-97d2-20fbbab7c05a.png)